.reek.yml

```diff
@@ -0,0 +1,3 @@
+detectors:
```

Comment:
need to add reek to dev dependencies and run it as part of CI ?

Reply:
It could be nice addition :)